modular_citadel/code/modules/reagents/chemistry/reagents/fermi_reagents.dm

```diff
@@ -0,0 +1,1596 @@
+ //Fermichem!!
+//Fun chems for all the family
+
+/datum/reagent/fermi
+	name = "Fermi" 	//Why did I putthis here?
+	id = "fermi"	//It's meeee
```

Comment:
as much as everyone likes you code comments should be a bit more professional and while i'm not sure about just forbidding things from being called fermichem (this is technically all a massive player reference but i'll have to have a staff discussion on it later) comments like these are very unnecessary.

Reply:
Ah, I put fermi because I thought it wouldn't ref me.

Reply:
It is partially based off Enrico Fermi though, kinda...!